### PR TITLE
handle alias activities in ProcessManifest

### PR DESCRIPTION
**Pull request by LouisonGitzinger**
Hi, I added the ability to take alias activities into account when fetching activities from the AndroidManifest.xml file.

I also created a test and a basic APK in the repo to test the feature.

Thanks, 

Louison

### CODE REVIEW

File: soot-infoflow-android/src/soot/jimple/infoflow/android/manifest/ProcessManifest.java [hunk 1]
```diff
@@ -629,4 +682,25 @@ else if (intentFilter.getTag().equals("category") && intentFilter.getAttribute("
 		return allLaunchableActivities;
 	}
 
-}
\ No newline at end of file
+	public void test() {
```
Comment (StevenArzt) [inline soot-infoflow-android/src/soot/jimple/infoflow/android/manifest/ProcessManifest.java]: What is this method for?

**Comment by StevenArzt** [comment]
Thanks a lot, that's a great merge request. I just wonder what the ```test()``` method is for. That method doesn't look like a part of the implementation, and the test case is separate anyway. Aside from that, the MR is ready for acceptance.

File: soot-infoflow-android/src/soot/jimple/infoflow/android/manifest/ProcessManifest.java (hunk 1 above)
Comment (LouisonGitzinger) [reply soot-infoflow-android/src/soot/jimple/infoflow/android/manifest/ProcessManifest.java]: Hello Steven, 
It was a test I forgot to delete for another feature I'm working on.
Let me fix that

**Comment by LouisonGitzinger** [comment]
@StevenArzt I deleted the test method, sorry for that.

**Pull request merged**